src/query/storage/index_test.go

```diff
@@ -122,17 +122,6 @@ func TestFetchQueryToM3Query(t *testing.T) {
 				},
 			},
 		},
-		{
-			name:     "regexp match dot plus -> field",
-			expected: "field(t1)",
-			matchers: models.Matchers{
-				{
-					Type:  models.MatchRegexp,
-					Name:  []byte("t1"),
-					Value: []byte(".+"),
-				},
-			},
-		},
```

Comment:
don’t we want to keep this test? we still want to special case this yeah? just not if there’s other input after

Reply:
Ah damn clumsy c/p job